src/components/pages/observation/list/views/stats/table-totals.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box, SimpleGrid, Text } from "@chakra-ui/react";
+import useTranslation from "@hooks/use-translation";
+import React from "react";
+
+export default function TableTotals({ totals }) {
+  const { t } = useTranslation();
+  return (
+    <SimpleGrid columns={{ md: 4 }} spacing={4} mb={4}>
+      {Object.keys(totals).map((metric) => (
```

Comment:
I didn't noticed full implementation earlier, concatination strings that way can be problemetic we should avoid it at all times unless absolutely necessary

- since this makes it hard to find when refactoring since key will not be present
- also they don't have underscores so it's harder to read and we should avoid it all times
- since it will become dynamic, it can not be statically compiled

possible solution might be have a render component that can do this easily like below, even full translated content can be passed to child row directly

```jsx
export default function TableTotals({ title ,count }) {
	return <tr>
		<td>{title}</td>
		<td>{count}</td>
	</tr>
}
```

and then use it like this

```jsx
<TableTotals title={t("STATS_BAR.TOTAL_IDENTIFIERS")} count={counts["identifiers"]}/>
<TableTotals title={t("STATS_BAR.TOTAL_TAXA")} count={counts["taxa"]}/>
```

p.s. we have this at sevral places in earlier code (written by me before I knew implications) and we might do a refactor to tackle this issue in future